templates/admin/user.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base_private.html" %}
+
+{% from "components/alert.html" import Alert %}
+{% from "components/delete_confirmation.html" import DeleteConfirmation %}
+{% from "components/icon.html" import Icon %}
+{% from "components/modal.html" import Modal %}
+
+{% block content %}
+  <div class="ccpo-panel-container">
+    <div class='col'>
+      <br>
+      <div class="h2">
+        User: {{ user.displayname }} ( {{ user.dod_id }} )
+      </div>
+
+      {% include "fragments/flash.html" %}
+
+
+    </div>
+
+    <pre>
+      user:
+      {{ user.displayname }}
+      user:
+      {{ user.email }}
+    </pre>
```

Comment:
I know there aren't any requirements for this page at this time; however, I'd like to try to present something a little more polished than the word "user:" with two different fields. Perhaps we could do something like:

```suggestion
    <p>{{ user.displayname }} &lt;{{ user.email }}&gt;</p>
```

or even 
```suggestion
    <pre>
      Name:
      {{ user.displayname }}
      Email Address:
      {{ user.email }}
    </pre>
```